packages/@react-stately/layout/src/TableLayout_DEPRECATED.ts

```diff
@@ -0,0 +1,426 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ColumnProps, TableCollection} from '@react-types/table';
+import {GridNode} from '@react-types/grid';
+import {Key} from 'react';
+import {LayoutInfo, Point, Rect, Size} from '@react-stately/virtualizer';
+import {LayoutNode, ListLayout, ListLayoutOptions} from './ListLayout';
+
+
+type TableLayoutOptions<T> = ListLayoutOptions<T> & {
+  getDefaultWidth: (props) => string | number
+}
+
+export class TableLayout_DEPRECATED<T> extends ListLayout<T> {
```

Comment:
I would do this the over way around and name the "new" hooks with a different name so that it isn't a breaking change for existing hooks.
I understand that the swap over would be easier when no longer under feature flag, but I'd rather avoid renaming the existing hooks if we can.

Reply:
I will do whatever you want but in my experience when doing "NEW" or "V2", the old gets forgotten about and is harder to untangle in the future. Are you _sure_ you would like me to make that change? (Again, i'll do whatever you want but I just want to make absolutely sure it's what you want)

Reply:
leaving for now